gzip/README.md

```diff
@@ -0,0 +1,31 @@
+# gzip 
+
+Build the [gzip](https://www.gnu.org/software/gzip/) data compressor as a
```

Comment:
Please use one sentence per line and one line per sentence, since it's easier to review and the github markdown renderer will space it properly.